service_worker.patch

```diff
@@ -1,16 +1,16 @@
 diff --git a/app/service-worker.js b/app/service-worker.js
-index 3b8fa90..835d200 100644
+index 0d3cd0f1..ce105510 100644
 --- a/app/service-worker.js
 +++ b/app/service-worker.js
-@@ -13,7 +13,16 @@ self.addEventListener('install', event =>
+@@ -13,7 +13,16 @@ self.addEventListener('install', (event) =>
  );
- 
- self.addEventListener('fetch', event => {
+
+ self.addEventListener('fetch', (event) => {
 -    event.respondWith(
--        caches.match(event.request.url).then(response => response || fetch(event.request))
+-        caches.match(event.request.url).then((response) => response || fetch(event.request))
 -    );
 +    if( event.request.url.indexOf("keeweb?config") == -1 )
-+    {   
++    {
 +        event.respondWith(
 +            fetch(event.request)
 +        );
```

Comment:
Let's drop the patch if it's no longer applied?

Reply:
If no one tells us what the server worker is needed for - yes, sure.